stubs/pyquaternion.pyi

```diff
@@ -0,0 +1,16 @@
+"""Type stubs for PyQuaternion."""
+
+from typing import List, Tuple
+
+class Quaternion:
+
+    # Note __init__ has many possible constructors. This is the one we use.
+    def __init__(self, axis: Tuple[float, float, float], angle: float): ...
```

Comment:
```suggestion
    def __init__(self, *, axis: Tuple[float, float, float], angle: float): ...
```

since I think all arguments to `Quarternion` constructors have to be specified as keyword arguments to avoid ambiguity (or at least I think we should enforce this in our code).